doc/langdef.md

```diff
@@ -2070,7 +2070,8 @@ See [cel-go/issues/9](https://github.com/google/cel-go/issues/9).
       (double) -> int
     </td>
     <td>
-      type conversion
+      Type conversion. Rounds toward zero, then errors if result is out of
+      range. See also round().
```

Comment:
Let's introduce `round`, `floor`, and `ceil` into a `math` namespace in a separate PR. 
e.g. `math.round(double)`, etc.

Reply:
Done.